packages/domain-picker/src/domain-picker/index.tsx

```diff
@@ -45,6 +46,15 @@ export const ItemGrouper: FunctionComponent< {
 	return <>{ children }</>;
 };
 
+export const ItemGroupLabel: FunctionComponent = function ItemGroupLabel( { children } ) {
+	return <p className="domain-picker__suggestion-group-label">{ children }</p>;
+};
+
+// For free sub-domains we mock the DomainSuggestion object and set the price as an empty string ''
+function domainIsFree( suggestion: DomainSuggestion ): boolean {
```

Comment:
Can't we use the `isFree` property from [types.ts#L123](https://github.com/Automattic/wp-calypso/blob/update/focused-launch/default-domain-last-in-picker/packages/data-stores/src/domain-suggestions/types.ts#L123)?

Reply:
It's an optional property which currently does not get returned from the mockDomainSuggestions function but we could add it to that to always return `true` as it does only get used for free sub-domains